tests/instruments/tdk/test_tdk_base.py

```diff
@@ -50,7 +50,8 @@ def test_multidrop_capability():
             [(b"ADR 6", b"OK"),
              (b"MDAV?", b'1')]
     ) as instr:
-        assert instr.multidrop_capability == True
+        value = instr.multidrop_capability
+        assert isinstance(value, bool) and value
```

Comment:
```suggestion
        assert instr.multidrop_capability is True
```
If you want to make a comparison with a boolean (or with None), use `is`. Your test is overly complicated.